### PR TITLE
Fix all but one test

### PR DESCRIPTION
**Pull request by raman325**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:

### CODE REVIEW

**Comment by raman325** [comment]
@firstof9 might need you to spend some time figuring out this last one, I'm at a loss

**Comment by firstof9** [comment]
Ooof looks like github's having trouble with openzwave...

**Comment by firstof9** [comment]
Try adding `sudo apt-get install libudev-dev` before the `pip install` in the work flow

**Comment by firstof9** [comment]
Well found the right `zwave_js_server` it's looking for, but the tests fail due to the code changes, we're kinda at the mercy of the pytest plugin, we'll have to wait for 2021.10.0b0 to drop before we can properly fix the tests right now.

**Comment by firstof9** [comment]
It's bailing on mqtt missing `Messages`

**Comment by firstof9** [comment]
Try changing line `29-36` of `tests/common.py`:

from:
```python
    with patch("homeassistant.components.mqtt.MQTT", return_value=AsyncMock()):
        mqtt_entry = MockConfigEntry(
            domain="mqtt",
            title="mqtt",
            data={CONF_BROKER: "http://127.0.0.1", CONF_PORT: 1883},
        )
        mqtt_entry.add_to_hass(hass)
        assert await hass.config_entries.async_setup(mqtt_entry.entry_id)
```

to:
```python
    mqtt_entry = MockConfigEntry(
        domain="mqtt", state=config_entries.ConfigEntryState.LOADED
    )
    mqtt_entry.add_to_hass(hass)
```

Let's see if that solves it.

File: tests/conftest.py [hunk 1]
```diff
@@ -18,6 +19,11 @@
 pytest_plugins = "pytest_homeassistant_custom_component"
 
 
+@pytest.fixture(autouse=True)
+def auto_enable_custom_integrations(enable_custom_integrations):
+    yield
```
Comment (firstof9) [inline tests/conftest.py:24]: 22-24 are duplicate lines of 281-283

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
Looks like the tests are fixed by the magic of updated upstream dependencies 🎉

**Pull request merged**